packages/osd-optimizer/postcss.config.js

```diff
@@ -29,5 +29,7 @@
  */
 
 module.exports = {
-  plugins: [require('autoprefixer')()],
+  plugins: [
+    /*require('autoprefixer')()*/
```

Comment:
I will be raising a PR on `autoprefixer` to silent their warning when they are not needed via an option flag. They wrongly assume that not being needed today would make them not needed in the future.

Today, they generate a ton of warnings about being not needed, masking important log messages.

Reply:
A little more context here would help, not a blocker but where are these warnings generated? Also cant we just wait for the PR that fixes auto prefixer?

Reply:
`autoprefixer` is a PostCSS plugin belonging to the days when browsers has vendor prefixes for rules. The revised browser matrix makes multi-vendor prefixing for all the _existing_ prefixes obsolete and `autoprefixer` throws a warning for every single time it is sees a prefix saying as much.

However, what `autoprefixer` fails to remember is that _"existing"_ is not future-proof and any day, we could have a new rule created that is thrown behind vendor prefixes.

The change I will be proposing will allow consumers of `autoprefixer` to silent this warning using an option. Until that change is made, merged, and released, I am stopping the use of `autoprefixer`. However, I am not removing it completely because we would want it back as soon as we have an update.